x-pack/plugins/fleet/public/components/enrollment_instructions/manual/index.tsx

```diff
@@ -6,8 +6,10 @@
  */
 
 import React from 'react';
+
```

Comment:
This change triggered with linting. I'm assuming this has already been discovered by the team, and if I merge upstream before merging the PR, it'll most likely have already been resolved.